### ISSUE TITLE
View Summary Stats for search results

### ISSUE THREAD

**Issue by dvenprasad** [AlexsLemonade/refinebio#438]
### Context

Results of usability evals
### Problem or idea

Users wanted to see summary stats for their search results so that they could quickly explore the data available before diving in. 

### Solution or next step

This needs to be discussed to determine feasibility and design post-mvp

**Comment by Miserlou** [comment]
Can you give an example of what that means.

**Comment by jaclyn-taroni** [comment]
In my mind, this is accomplished with the filters -- the number for each species, etc. is displayed there. If user want more specific information like _disease_ or _tissue_, this is probably not feasible in the absence of normalized metadata.

Was this comment more about visualization with something like an interactive pie chart @dvenprasad ?

**Comment by dvenprasad** [comment]
Yes, it's more along the lines of visually being able to view the summary stats explore the data that way. 

It's not necessary for the Sept 1 release.
